server/meshmodel/istio-base/1.16.0/v1.0.0/relationships/hierarchical_inventory_WASMFilterToEnvoyPlugin.json

```diff
@@ -0,0 +1,73 @@
+{
+    "schemaVersion": "core.meshery.io/v1alpha2",
```

Comment:
I propose that we move this to:

```
"schemaVersion": "core.meshery.io/v1alpha2",

Reply:
Surely, @leecalcote I saw the recent changes made by Uzair to the schemaVersion I'll be updating those refs.